### ISSUE TITLE
Mobile Menu UX

### ISSUE THREAD

**Issue by gruppler** [The1Rogue/Attak#3]
When the menu contents are too tall for the screen, I'm unable to scroll. It's also a little confusing since it's not immediately clear which sections are expanded or collapsed.

**Comment by The1Rogue** [comment]
Yes, the entire menu system was not working, as I wanted to add more, it has gotten too much on desktop too in my opinion. So a full menu overhaul is already in the works.
